packages/next/src/build/swc/generated-native.d.ts

```diff
@@ -38,18 +38,18 @@ export interface TransformOutput {
   map?: string
   output?: string
 }
-export declare function mdxCompile(
```

Comment:
Revert this file? Hopefully https://github.com/vercel/next.js/pull/74654 fixes whatever was causing this to be regenerated?